packages/calcite-components/src/components/table-row/table-row.scss

```diff
@@ -5,11 +5,9 @@
  *
  * @prop --calcite-table-row-background: [Deprecated] Use `--calcite-table-row-background-color` instead. Specifies the background color of the component.
  * @prop --calcite-table-row-background-color: Specifies the background color of the component.
- * @prop --calcite-table-row-background-color-striped: The background color of the component's `striped` rows, when specified.
+ * @prop --calcite-table-row-secondary-background-color: The background color of the component's `striped` rows, when specified.
```

Comment:
I discussed this token name change with Skye and Aaron. The use of the "secondary" group here aligns better with our naming schema.

Reply:
Hm. This seems unexpected to me. The name corresponds directly to the property that causes it to display. We don’t really use secondary elsewhere.

Reply:
“striped-background-color”?

Reply:
primary / secondary is a common naming pattern for design system colors and one I think we'll be expanding on in the future. "striped" is very specific to table.

Reply:
Yeah, I guess that's my take - it is specific to table. To this point we've been intentional to *not* use primary / secondary / tertiary, so teams can build their own meaning through our properties. I think that's a fine direction to head in if we choose to in the future, but beginning to add it in now seems strange - this css property can only be used when `striped` is used so I think it makes sense. Adding "secondary" verbiage as the only public-facing case of it here seems strange. Maybe a good discussion with the larger team.

Reply:
does "--calcite-table-row-striped-secondary-background-color" work? That way at least there is some indication to a user it pertains and will only be visible when the `striped` property is in use.

Reply:
I would agree with this. If the default row color is "--calcite-table-row-background-color", then I would expect the striped prop to correlate to a "--calcite-table-row-striped-background-color" css property. I don't think the word "secondary" is necessary especially since there is no need for a "--calcite-table-row-primary-background-color" for it to correlate to.